### PR TITLE
Render monospaced table in HTML when not curl request

### PR DESCRIPTION
**Pull request by archcorsair**
Addresses #21 
- If the user agent is not a curl request we respond with some formatted HTML
- Added `--inspect` flag to nodemon to allow using chrome dev tools debugging

### CODE REVIEW

**Comment by archcorsair** [comment]
This is difficult for me to test locally since the color codes are not stripped.

**Comment by sagarkarira** [comment]
Thanks, I will review and merge when I get time.
I used [strip-ansi](https://www.npmjs.com/package/strip-ansi) module

File: app.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ function errorHandler(error, res) {
 app.use(morgan(':remote-addr :remote-user :method :url :status :res[content-length] - :response-time ms'));
 
 app.get('/', (req, res) => {
+  const isCurl = req.headers['user-agent'].match(/\bcurl\b/gmi) !== null
```
Comment (hobroker) [inline app.js]: I'd suggest to check for `text/html` instead of `curl`

**Comment by sagarkarira** [comment]
@archcorsair I added the HTML template in countries API too. Also added black background and green color. 

The table top line is extending far down the page. I was not able to figure out why it's happening. I can take a look at that later. For now, I merging this. 

Thanks for your PR. 
😃

**Pull request merged**